src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -199,15 +199,14 @@
   "Returns a boolean indicating whether or not the given certname exists in the db"
   [certname]
   {:pre [certname]}
-  (sql/with-query-results result-set
-    ["SELECT 1 FROM certnames WHERE certname=? LIMIT 1" certname]
-    (pos? (count result-set))))
+  (not (empty? (jdbc/query ["SELECT 1 FROM certnames WHERE certname=? LIMIT 1"
```

Comment:
@ajroetker hmm, I believe query returns a (possibly empty) vector, not nil, so we may need empty? here.